src/services/token.service.js

```diff
@@ -0,0 +1,29 @@
+const { Token } = require('../models/Token.model');
+
+const save = async (userId, newToken) => {
+  const token = await Token.findOne({ where: { UserId: userId } });
```

Comment:
Ensure that the 'UserId' field matches the actual field name in your database schema. Sequelize typically uses camelCase for field names unless explicitly defined otherwise.